core/src/components/picker-column-internal/picker-column-internal.scss

```diff
@@ -0,0 +1,54 @@
+@import "../../themes/ionic.globals";
+
+// Picker Internal
+// --------------------------------------------------
+
+:host {
+  @include padding(0px, 16px, 0px, 16px);
+
+  height: 200px;
+
+  outline: none;
+
+  font-size: 22px;
+  scroll-snap-type: y mandatory;
+
+  /**
+   * Need to explicitly set overflow-x: hidden
+   * for older implementations of scroll snapping.
+   */
+  overflow-x: hidden;
+  overflow-y: scroll;
+
+  // Hide scrollbars on Firefox
+  scrollbar-width: none;
+}
+
+/**
+ * Hide scrollbars on Chrome and Safari
+ */
+:host::-webkit-scrollbar {
+  display: none;
+}
+
+:host .picker-item {
+  height: 34px;
```

Comment:
My thought is having a picker item wrap onto 2 lines is bad UX and not something we should encourage. I do not see any native picker usages on iOS where the items wrap onto 2 lines. Thoughts?

Reply:
Makes sense. Maybe this should just cut the text off with an ellipsis, then, instead of wrapping? It looks pretty broken if items in the middle of the list wrap to the next line, since the wrapped text overlaps the item below.